platform/commonUI/edit/src/objects/EditableDomainObject.js

```diff
@@ -102,6 +103,15 @@ define(
                             capability;
                 };
 
+
+                editableObject.setOriginalObject = function(object) {
+                    originalObject = object;
+                };
+
+                editableObject.getOriginalObject = function() {
```

Comment:
Ignore this. Needed a way of retrieving the non-editable version of an object from the editable version, but I'm going to refactor this out so that it's (probably) not needed.